### PR TITLE
[13.0][MIG] account_document_reversal, migration to 13.0

### PR DESCRIPTION
**Pull request by kittiu**
Migration with some conceptual changes (because no more account.inovice).
Please read on the README.

Ready for review.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
It seems the module https://github.com/OCA/account-financial-tools/tree/13.0/account_document_reversal is already available in v13 :astonished:

**Comment by kittiu** [comment]
> It seems the module https://github.com/OCA/account-financial-tools/tree/13.0/account_document_reversal is already available in v13

Nop, just today that we are having this problem. The one I accidentally merged 12 days ago, just seem to be auto merged (so, what is merged now was the unfinished product).
We are just discussing it, https://github.com/OCA/account-financial-tools/pull/956

**Comment by kittiu** [comment]
@AaronHForgeFlow back to green now :)

**Comment by kittiu** [comment]
@Saran440 please help review if you have time. Thanks!

**Comment by Saran440** [comment]
Functional Test :+1: 
if you check cancel reversal in Payment menu should hide reset to draft.
 
Selection_001

**Review by JordiBForgeFlow — APPROVED** [review]

File: account_document_reversal/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Account Document Reversal",
+    "summary": "Create reversed journal entries when cancel document",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft," "Eficent," "Odoo Community Association (OCA)",
```
Comment (JordiBForgeFlow) [inline account_document_reversal/__manifest__.py]: @kittiu can you change Eficent -> ForgeFlow?

Comment (kittiu) [reply account_document_reversal/__manifest__.py]: Done

**Comment by JordiBForgeFlow** [comment]
cannot merge due to fail in pre-commit https://github.com/OCA/account-financial-tools/runs/4773419136?check_suite_focus=true#step:4:138. Development_status = "Beta" not allowed?¿ @pedrobaeza Can you help?

**Comment by pedrobaeza** [comment]
Not sure, but I would remove the development_status key, as it's the default one.

**Comment by JordiBForgeFlow** [comment]
@kittiu if you proceed to remove the development_status I can try to merge again.

**Comment by kittiu** [comment]
> @kittiu if you proceed to remove the development_status I can try to merge again.

Removed, shall you try again please.

**Pull request merged**